context.go

```diff
@@ -583,24 +585,57 @@ func (ctx Context) Logger(module ...Module) *zap.Logger {
 	return ctx.cfg.Logging.Logger(mod)
 }
 
+type slogHandlerFactory func(handler slog.Handler, core zapcore.Core, moduleID string) slog.Handler
+
+var (
+	slogHandlerFactories   []slogHandlerFactory
+	slogHandlerFactoriesMu sync.RWMutex
+)
```

Comment:
I know this is an annoying request, but how would you feel if we drop the "factory" thing? 😆 I just... it feels like Java.

What about like `slogHandlerWrapper` instead? Or, maybe even just `slogHandler`? I am not 100% sure since I haven't dived into slog too much yet to be that familiar with it.